### DISCUSSION TITLE
I want to report a repository full of junk content

### DISCUSSION THREAD

**Discussion by NianBroken** [community/community#57755] [Code Security]
### Select Topic Area

Bug

### Body

"Https://github.com/cirosantilli/china-dictatorship" in the warehouse is full of useless junk information, the contents of this warehouse inside contains all known Chinese characters, which leads to no matter what I search for content, This garbage warehouse will appear on my computer screen and interfere with my work. The bad impact of this warehouse on me is immeasurable, I ask the Github official to block and delete this warehouse and the authors of this warehouse.

**Comment by cirosantilli** [comment]
Truly terrible repo.
